inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -673,7 +642,7 @@
                         "type": "string"
                     },
                     "version": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
@kaplun: this change introduces this error: https://travis-ci.org/inspirehep/inspire-next/jobs/132557014#L6089